src/main/webapp/js/userMap.js

```diff
@@ -1,13 +1,19 @@
+var countryCoordinates;
```

Comment:
Not entirely sure about how this works, but is this really needed? Can we just pass the variable to the function itself to reduce global namespace pollution?